raiden_contracts/tests/test_token.py

```diff
@@ -60,8 +67,12 @@ def test_token_transfer_funds(web3, custom_token, get_accounts, txn_gas):
 
 
 def test_custom_token(custom_token, web3):
-    assert web3.eth.getCode(custom_token.address)
+    """ See custom_token.address contains some big code """
+
+    assert len(web3.eth.getCode(custom_token.address)) > 100
```

Comment:
Why care about the size of the code?

Reply:
In case of empty, '0x' is returned.  `assert '0x'` silently succeeds.

Reply:
Ah good point @pirapira

Reply:
In that case you should also change the docstring though -- as anyone who reads the code will have the same question I had.

Reply:
shouldn't it decode the hex representation and check the length is larger then 0?

Reply:
Maybe we also have the exact expected hex string.